packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -259,6 +269,9 @@ export class ModularChangeFamily
 			if (change.change.fieldChanges !== undefined) {
 				fieldChanges.push(tagChange(change.change.fieldChanges, change.revision));
 			}
+			if (change.change.valueConstraint !== undefined) {
```

Comment:
If we are composing multiple value constraints on this node, I think we should only use the first one, while this implementation uses the last one.

Also, if we're composing a value change with a later value constraint, I think we should drop the value constraint.

Reply:
Switched to using the first constraint in latest commit, but not sure I completely understand what you mean in the second part.